### ISSUE TITLE
Some top menus not working in Safari on iPad

### ISSUE THREAD

**Issue by MartinCRichards** [smurphboy/OARCLogger#26]
The More and WAOARC menus don’t drop down with Safari on iPad; other menus are ok

**Comment by smurphboy** [comment]
Agreed. I replicated on my iPad mini. All working in Chrome though. Will investigate.
Workaround is to turn into portrait where the menu is fully expanded, then you can access the sub-menu items.
